pkgs/applications/audio/faust/faust2.nix

```diff
@@ -194,8 +196,8 @@ let
         # export parts of the build environment
         for script in "$out"/bin/*; do
           wrapProgram "$script" \
-            --set FAUSTLIB "${faust}/lib/faust" \
-            --set FAUST_LIB_PATH "${faust}/lib/faust" \
+            --set FAUSTLIB "${faust}/share/faust" \
```

Comment:
FAUSTLIB is mainly (afaik) for .cpp files that the compiler combines with the generated code to create the final product.